java/system-test/system-test-suite/src/test/java/sleeper/systemtest/suite/QueryIT.java

```diff
@@ -177,4 +177,67 @@ void shouldRunQueryWithTwoRangesThatDoNotOverlap(SleeperSystemTest sleeper) {
                             LongStream.range(10, 20), LongStream.range(30, 40))));
         }
     }
+
+    @Nested
+    @DisplayName("WebSocket query")
+    class WebSocketQuery {
```

Comment:
Can we test a query that returns no results please?

Reply:
Done, thanks